frontend/www/js/omegaup/arena/arena.ts

```diff
@@ -543,6 +538,93 @@ export class Arena {
       });
     }
 
+    // Setup run submit view, if it is available.
+    if (document.getElementById('run-submit') !== null) {
+      this.runSubmitView = new Vue({
+        el: '#run-submit',
+        render: function(createElement) {
+          return createElement('omegaup-arena-runsubmit', {
+            props: {
+              languages: this.languages,
+              submissionGapSecondsRemaining: this.submissionGapSecondsRemaining,
+              preferredLanguage: this.preferredLanguage,
+            },
+            on: {
+              'submit-run': (code: string, language: string) => {
+                this.submitRun(code, language);
+              },
+            },
+          });
+        },
+        methods: {
+          submitRun: (code: string, language: string): void => {
+            const problemset = this.computeProblemsetArg();
+
+            api.Run.create(
+              $.extend(problemset, {
```

Comment:
nit:

```suggestion
              Object.assign(problemset, {
```

hay que intentar no usar jQuery en código tocado. y puedes agregar el [polyfill](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/assign#Polyfill) a `frontend/www/js/omegaup/polyfills.js`?

Reply:
Listo.